packages/shared/world/SceneWorker.ts

```diff
@@ -182,7 +184,7 @@ export class SceneWorker {
       this.ready |= SceneWorkerReadyState.DISPOSED
     }
 
-    getUnityInstance().UnloadScene(this.loadableScene.id)
+    getUnityInstance().UnloadScene(this.loadableScene.sceneNumber)
```

Comment:
How should we keep this kernel update retro-compatible with the `dev` branch renderer using scene id? ...